alerting/src/main/kotlin/org/opensearch/alerting/AlertService.kt

```diff
@@ -190,6 +191,19 @@ class AlertService(
             )
         }
 
+        // Including a list of triggered clusters for cluster metrics monitors
+        var triggeredClusters: MutableList<String>? = null
```

Comment:
Consider this is for experiemental, does here need a check for cluster setting for this feature?

Reply:
I don't think that would be necessary, as the feature disabled, this will always be just the local cluster. But I'm open adding a setting check if we think that's best.